pyproject.toml

```diff
@@ -58,28 +58,27 @@ progressbar2 = "^3.55.0"
 PyYAML = ">=5.1,<7.0"
 scikit-learn = "^1.0.1"
 scipy = "^1.7.2"
-SQLAlchemy = "<1.4.0,>=1.3.0"
+SQLAlchemy = "^1.4,<1.5"
 numpy = ">=1.22.0,<2.0"
 jinjasql = "^0.1.8"
-ipython = "^7.29.0"
+ipython = ">=8.0.1"
```

Comment:
Motivated not by sqlalchemy but by vulnerability issue.

https://www.cvedetails.com/cve/CVE-2022-21699/

https://ipython.readthedocs.io/en/stable/whatsnew/version8.html#ipython-8-0-1-cve-2022-21699